src/utils/server-routes/index.js

```diff
@@ -0,0 +1,11 @@
+export { performUserSignup } from "./server-routes";
+export { performUserLogin } from "./server-routes";
+export { performGetCategories } from "./server-routes";
+export { performGetProducts } from "./server-routes";
+export { performAddToCart } from "./server-routes";
+export { performGetCart } from "./server-routes";
+export { performUpdateCartQuantity } from "./server-routes";
+export { performRemoveFromCart } from "./server-routes";
+export { performAddToWishlist } from "./server-routes";
+export { performGetWishlist } from "./server-routes";
+export { performRemoveFromWishlist } from "./server-routes";
```

Comment:
```suggestion
export { performUserSignup, performUserLogin, performGetCategories, performGetProducts , performAddToCart , performGetCart  , performUpdateCartQuantity ,performRemoveFromCart , performAddToWishlist, performGetWishlist    ,performRemoveFromWishlist } from "./server-routes";
```

Reply:
Did not know of the implications of having multiple exports, got to learn a new concept!